pages/_lang/generate.vue

```diff
@@ -1118,7 +1135,7 @@ footer a {
   padding-bottom: 41px;
 }
 #screenshotDownloadButton {
-  width: 156px;
+  width: 174px;
```

Comment:
@amrutha95 why are we specifying an explicit width?

Reply:
Since I changed the text I had to increase the width. Could be fit-content instead too.

Reply:
Actually I remember why I chose an explicit width now. If I make it fit-content, then when it's clicked on and the loader appears, the button resizes.